Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -1,458 +1,429 @@
 import UIKit
 import Result
 
-public final class DiskMemoryPersistenceStack: NSObject, PersistenceStack {
+public extension Persistence {
 
-    public struct Configuration {
+    public final class DiskMemoryPersistenceStack: NSObject, PersistenceStack {
```

Comment:
With this namespace, we end up having `Persistence.DiskMemoryPersistenceStack`, repeating twice the `Persistence` bit. Should we rename the class to `DiskMemoryStack`, which results in `Persistence.DiskMemoryStack`?

Reply:
Also thought about it, but we have the same "issue" on `Network.URLSessionNetworkStack`

The thing is that it's a `PersistenceStack`, so it makes sense for it to be a `SomethingPersistenceStack`.

But yes, I agree that it gets a bit weird now with the namespace 😆

Still not convinced with `Persistence.DiskMemoryStack` though 🤔